packages/use-contractkit/src/screens/cew.tsx

```diff
@@ -8,10 +8,15 @@ import { ConnectorProps } from '.';
 export const CeloExtensionWallet: React.FC<ConnectorProps> = ({
   onSubmit,
 }: ConnectorProps) => {
-  const { network, initConnector, initError: error } = useContractKitInternal();
+  const {
+    network,
+    initConnector,
+    initError: error,
+    feeCurrency,
+  } = useContractKitInternal();
 
   const initialiseConnection = useCallback(async () => {
-    const connector = new CeloExtensionWalletConnector(network);
+    const connector = new CeloExtensionWalletConnector(network, feeCurrency);
     await initConnector(connector);
     void onSubmit(connector);
   }, [initConnector, network, onSubmit]);
```

Comment:
probably good to remove the lint warning here by adding the `feeCurrency` to the depsArray